email_templates/html/decom_email.html.j2

```diff
@@ -25,9 +26,9 @@ Where possible we will not decommission hardware that is still hosting User owne
 if the<br>Hypervisor hosting your VM fails we have NO spares and will NOT attempt anything beyond basic repair work,
 <br>in this case you will suffer the total loss of your VM.<br><br>
 
-Failure to comply with this notice will eventually lead to Hardware being turned off and disposed of, whether it<br>
-is still hosting VMs or not, eventually we will have no choice. Routine reminders will follow and increase in
-frequency over the coming weeks.<br><br>
+Failure to comply with this notice by mid day (12:00PM) on Wednesday 28th February 2024 will lead to Hardware being 
```

Comment:
One for the future but we should add a variable like `{{ Time }}` and `{{ Date }}` to make this template more generic

Reply:
Yeh, was thinking something similar given that we'll be doing this for at least 1 generation per year but yes - change for down the road